nvtabular/dask/io.py

```diff
@@ -249,7 +262,12 @@ def __init__(self, *args, row_groups_per_part=None):
         if row_groups_per_part is None:
             # TODO: Use `total_byte_size` metadata if/when we figure out how to
             #       correct for apparent dict encoding of cat/string columns.
-            path0 = self.fs.sep.join([self._base, self._metadata.row_group(0).column(0).file_path])
+            if os.path.isdir(self._base):
```

Comment:
Lets try to avoid using `os` here since we already have an fsspec object to work with (`self.fs.isdir()`)